allennlp/tests/semparse/domain_languages/quarel_language_test.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.semparse.domain_languages.quarel_language import QuaRel
```

Comment:
Add `QuaRel` to `domain_languages.__init__`, so you can import it here with `from allennlp.semparse.domain_languages import QuaRel`.  You also might consider calling it `QuaRelLanguage`, so that it's more obvious what it is when you instantiate it below (i.e., it's not a qualitative relationship object, it's the language).

Reply:
Also, our naming conventions are that the main class in a module should match the module name.  So if you want to stick with `QuaRel` as the name, I'd change the filename to `quarel.py` instead of `quarel_language.py`.